client/driver/docker.go

```diff
@@ -431,9 +443,9 @@ func (d *DockerDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle
 
 	config, err := d.createContainerConfig(ctx, task, d.driverConfig, syslogAddr)
 	if err != nil {
-		d.logger.Printf("[ERR] driver.docker: failed to create container configuration for image %s: %s", d.imageID, err)
+		d.logger.Printf("[ERR] driver.docker: failed to create container configuration for image %q: %v", d.driverConfig.ImageName, err)
```

Comment:
So should I also create the container using the Image ID instead of the Image Name? I'm worried about weird races between concurrent pulls of "latest" images causing the image ID to change between Prestart/PullImage and Start/CreateContainer... not sure if that's a realistic fear or not.